gradle-multi-module/jmp-cloud-bank-impl/src/main/java/module-info.java

```diff
@@ -0,0 +1,5 @@
+module jmp.bank.impl {
+  requires static lombok;
+  requires transitive jmp.bank.api;
```

Comment:
`jmp.bank.api` is direct dependency, `jmp.dto` will be transitive (since `jmp.bank.api requires jmp.dto`)

Reply:
Fixed